abstract_specification.md

```diff
@@ -263,7 +263,7 @@ Parameters:
 | read                                    | Read a subset of data from the SOMADataFrame                                   |
 | write                                   | Write a subset of data to the SOMADataFrame                                    |
 
-A SOMADataFrame is indexed by one or more dataframe columns (aka "dimensions"). The name and order of dimensions is specified at the time of creation. Slices are addressable by the user-specified dimensions. The `soma_joinid` column may be specified as an index column.
+A SOMADataFrame is indexed by one or more dataframe columns (aka "dimensions"). The name and order of dimensions is specified at the time of creation. Slices (see the _Slicing_ section below) are addressable by the user-specified dimensions. The `soma_joinid` column may be specified as an index column.
```

Comment:
I would make these hyperlinks. Up to you exactly how you do it but here is one option.

```suggestion
A SOMADataFrame is indexed by one or more dataframe columns (aka "dimensions"). The name and order of dimensions is specified at the time of creation. [Slices](#slicing) are addressable by the user-specified dimensions. The `soma_joinid` column may be specified as an index column.
```